.docker/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.4'
```

Comment:
Looks like this compose file is used for dev env, so better rename it `docker-compose.dev.yml`.

Reply:
You can define a `COMPOSE_FILE=.docker/docker-compose.dev.yml` env var in the `.env.dist`  file, which will be used to specify which compose file to use.

Reply:
Isn't that compose file needed for production too ? :thinking:

Reply:
then define a prod compose file : `docker-compose.prod.yml` ;)